script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -350,7 +350,7 @@ Those words were really scary.
 I wasn't sad because I might've been killed, but because I didn't want this person to have to do such a thing to me.
 <Page98>
 "Don't cry, Shiki. I won't apologize either.
-I just want you to remember that you did something wrong.
+I just want you to take to heart what wrong you did here.//alt suggestion: I want you to remember what you've done.
```

Comment:
Reads a bit stiff but take to heart is a good phrase.

Reply:
would "I just want you to take to heart what you did wrong" be less stiff and remain accurate?

Reply:
I just want you to take to heart what you did wrong here